x-pack/plugins/ml/public/application/explorer/swimlane_container.tsx

```diff
@@ -406,22 +406,29 @@ export const SwimlaneContainer: FC<SwimlaneProps> = ({
         <EuiFlexGroup
           gutterSize={'none'}
           direction={'column'}
-          style={{ width: '100%', height: '100%', overflow: 'hidden' }}
           ref={resizeRef}
           data-test-subj={dataTestSubj}
+          css={{
```

Comment:
I am seeing the swim lane actions menu switching to being left aligned. Can you move it back to the right?

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/7405507/180760980-92ab6f67-1f07-491e-bfd4-5c4ba20229fc.png">